### DISCUSSION TITLE
einkbro on Ratta SuperNote

### DISCUSSION THREAD

**Discussion by enomis123** [plateaukao/einkbro#200] [General]
Hi, 
I'd love to use EinkBro on my Supernote from Ratta. I am aware that this is not officially supported but a some very smart persons found a way to sideload apps to the device. EinkBro (9.16.0) runs on the device but unfortunately  "save to ePub/PDF" is not working ( I also tried older versions such as 8.xxx with no luck). 
The device runs Android 8.1 . Do you have any idea what is needed to make "save to ePub/pdf" run or which older version might be better compatible with Android 8.1 ( and still supports save to ePub)  ? 
Thank you very much for your great work !

**Comment by plateaukao** [comment]
@enomis123 
Glad to see that someone make EinkBro available on Supernote.
* For saving web content into pdf, it needs devices' support. If the device does not come with Google's print to pdf module, then it won't work.
* As for saving web content to epub, I also use system built-in file picker to choose where to save the epub. If the device does not come with a built-in file picker that is compatible with Google's original one, then it might not work. You could try to install several popular file explorers to see if they could provide the file picker feature as built-in one. If not, since I don't have Supernote, this is only my guess, and I couldn't look further into the issue.
